proto/serverless/proto/instrumentation/tags/v1/tags.proto

```diff
@@ -0,0 +1,295 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tags.v1;
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 100  //
+  // ========================================== //
+
+  // ============================================ //
+  // Top level tags will have their underscores replaced //
+  // with periods when flattened. So for `aws_lambda` it will //
+  // be converted to `aws.lambda`//
+
+  // The root AWS Lambda Span tags.
+  optional AwsLambdaTags aws_lambda = 100;
+
+  // The AWS Lambda Initialization tags.
+  optional AwsLambdaInitializationTags aws_lambda_initialization = 101;
+
+  // The AWS Lambda Invocation tags.
+  optional AwsLambdaInvocationTags aws_lambda_invocation = 102;
+
+  // The AWS SDK Tags.
+  optional AwsSdkDynamodbTags aws_sdk_dynamodb = 103;
+  optional AwsSdkSqsTags aws_sdk_sqs = 104;
+  optional AwsSdkSnsTags aws_sdk_sns = 105;
+
+
+  // The HTTP Tags.
+  optional HttpTags http = 106;
+
+  // The HTTPS Tags.
+  optional HttpTags https = 107;
+
+  // The Express Tags.
+  optional ExpressTags express = 108;
+}
+
+message SlsSdkTags {
+  // A Serverless Platform OrgId.
+  string org_id = 1;
```

Comment:
It'll be great to indicate by each tag, where we do we source information from, whether it's CW or Internal extension.

Reply:
Ok, we can go with such assumption. I'll then comment if I see one that cannot be set by extension, and yet it's not indicated it comes from CW

Reply:
Yeah I think that is the best path, once I finish the workflow for generating the docs we can see if we want to add more detail to make it clearer.